content/en/platform/corda/5.0/developing-applications/getting-started/first-flow/code-kotlin.md

```diff
@@ -302,7 +302,7 @@ We can now start sending messages to the responder:
    ```kotlin
            val response = session.receive(Message::class.java)
     ```
-    When the `send()` is sent to the responder, Corda executes the `MyFirstResponderFlow` responder flow `call()` method down to the `session.receive()`, which returns the payload that was sent in the initiator's `send()` method.
+   When the message is sent to the responder via the `send()` method, Corda executes the `session.receive()` in the `MyFirstResponderFlow` responder flow's `call()` method. This returns the payload that was sent in the initiator's `send()` method.
```

Comment:
I think we are mixing two terminology here - message and payload. They are the same thing.
And I still believe it's a bit complicated. What we want to say here is that what message is sent in the `send()` method is then retrieved using `session.receive()` method in the `MyFirstResponderFlow` responder flow's `call()` method.

So another suggestion:
When the message is sent to the responder via the `send()` method, Corda executes the `session.receive()` in the `MyFirstResponderFlow` responder flow's `call()` method. This `session.receive()` method retrieves the message that was sent in the initiator's `send()` method.
- and I would keep the message instead of payload as in the piece of code we have `Message::class.java`

Reply:
Updated according to your suggestion @anien